### DISCUSSION TITLE
How to hide Uno warnings?

### DISCUSSION THREAD

**Discussion by ArchieCoder** [unoplatform/uno#17552] [General]
When the app starts, there are a lot of warnings. 

1- For example, I find the FileStorage read package file warnings not really helpful. See section 1.

2- In App code, I set the CoreLogLevel to None and I still get ouput messages. See section 2.

**SECTION 1**
Uno.Extensions.Storage.FileStorage: Warning: ReadPackageFileAsync - File 'appsettings.keyvaluestorageconfiguration.json' does not exist in package
07-15 15:49:40.281-04:00 W/Uno.Extensions.Storage.FileStorage: ReadPackageFileAsync - File 'appsettings.keyvaluestorageconfiguration.json' does not exist in package 
Uno.Extensions.Storage.FileStorage: Warning: ReadPackageFileAsync - File 'appsettings.appconfig.json' does not exist in package
07-15 15:49:40.293-04:00 W/Uno.Extensions.Storage.FileStorage: ReadPackageFileAsync - File 'appsettings.appconfig.json' does not exist in package 
Uno.Extensions.Storage.FileStorage: Warning: ReadPackageFileAsync - File 'appsettings.localizationconfiguration.json' does not exist in package
07-15 15:49:40.296-04:00 W/Uno.Extensions.Storage.FileStorage: ReadPackageFileAsync - File 'appsettings.localizationconfiguration.json' does not exist in package 
Uno.Extensions.Storage.FileStorage: Warning: ReadPackageFileAsync - File 'appsettings.localizationsettings.json' does not exist in package
07-15 15:49:40.300-04:00 W/Uno.Extensions.Storage.FileStorage: ReadPackageFileAsync - File 'appsettings.localizationsettings.json' does not exist in package 
Uno.Extensions.Storage.FileStorage: Warning: ReadPackageFileAsync - File 'appsettings.navigationconfiguration.json' does not exist in package
07-15 15:49:40.303-04:00 W/Uno.Extensions.Storage.FileStorage: ReadPackageFileAsync - File 'appsettings.navigationconfiguration.json' does not exist in package 
07-15 15:49:40.324-04:00 I/Uno.Extensions.Navigation.Navigators.ContentControlNavigator: NavigateAsync - Starting Navigation - Navigator: ContentControlNavigator Request:  
Uno.Extensions.Navigation.Navigators.ContentControlNavigator: Warning: Show - Missing view for navigation path ''
07-15 15:49:40.333-04:00 W/Uno.Extensions.Navigation.Navigators.ContentControlNavigator: Show - Missing view for navigation path '' 
07-15 15:49:40.335-04:00 I/Uno.Extensions.Navigation.Navigators.ContentControlNavigator: CreateControlFromType - Creating control of type FrameView 
07-15 15:49:40.452-04:00 I/Uno.Extensions.Navigation.Navigators.FrameNavigator: NavigateAsync - Starting Navigation - Navigator: FrameNavigator Request: Main 
Microsoft.UI.Xaml.BrushTransition: Error: The member BrushTransition.BrushTransition() is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.BrushTransition.BrushTransition.BrushTransition%28%29
07-15 15:49:40.480-04:00 E/Microsoft.UI.Xaml.BrushTransition: The member BrushTransition.BrushTransition() is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.BrushTransition.BrushTransition.BrushTransition%28%29 
Microsoft.UI.Xaml.BrushTransition: Error: The member TimeSpan BrushTransition.Duration is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.BrushTransition.TimeSpan%20BrushTransition.Duration
07-15 15:49:40.484-04:00 E/Microsoft.UI.Xaml.BrushTransition: The member TimeSpan BrushTransition.Duration is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.BrushTransition.TimeSpan%20BrushTransition.Duration 
Microsoft.UI.Xaml.Controls.ContentPresenter: Error: The member BrushTransition ContentPresenter.BackgroundTransition is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Controls.ContentPresenter.BrushTransition%20ContentPresenter.BackgroundTransition
07-15 15:49:40.491-04:00 E/Microsoft.UI.Xaml.Controls.ContentPresenter: The member BrushTransition ContentPresenter.BackgroundTransition is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Controls.ContentPresenter.BrushTransition%20ContentPresenter.BackgroundTransition 
Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings: Error: The member ComboBoxTemplateSettings.DropDownContentMinWidth is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings.ComboBoxTemplateSettings.DropDownContentMinWidth
07-15 15:49:40.700-04:00 E/Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings: The member ComboBoxTemplateSettings.DropDownContentMinWidth is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings.ComboBoxTemplateSettings.DropDownContentMinWidth 
Microsoft.UI.Xaml.Media.Animation.SplitCloseThemeAnimation: Error: The member SplitCloseThemeAnimation.SplitCloseThemeAnimation() is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Media.Animation.SplitCloseThemeAnimation.SplitCloseThemeAnimation.SplitCloseThemeAnimation%28%29
07-15 15:49:40.733-04:00 E/Microsoft.UI.Xaml.Media.Animation.SplitCloseThemeAnimation: The member SplitCloseThemeAnimation.SplitCloseThemeAnimation() is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Media.Animation.SplitCloseThemeAnimation.SplitCloseThemeAnimation.SplitCloseThemeAnimation%28%29 
Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings: Error: The member ComboBoxTemplateSettings.DropDownOffset is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings.ComboBoxTemplateSettings.DropDownOffset
07-15 15:49:40.739-04:00 E/Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings: The member ComboBoxTemplateSettings.DropDownOffset is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings.ComboBoxTemplateSettings.DropDownOffset 
Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings: Error: The member ComboBoxTemplateSettings.DropDownOpenedHeight is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings.ComboBoxTemplateSettings.DropDownOpenedHeight
07-15 15:49:40.743-04:00 E/Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings: The member ComboBoxTemplateSettings.DropDownOpenedHeight is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings.ComboBoxTemplateSettings.DropDownOpenedHeight 
Microsoft.UI.Xaml.Media.Animation.SplitCloseThemeAnimation: Error: The member void Begin() is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Media.Animation.SplitCloseThemeAnimation.void%20Begin%28%29
07-15 15:49:40.746-04:00 E/Microsoft.UI.Xaml.Media.Animation.SplitCloseThemeAnimation: The member void Begin() is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Media.Animation.SplitCloseThemeAnimation.void%20Begin%28%29 

**SECTION 2**

07-15 16:26:35.746-04:00 W/Uno.Extensions.Storage.FileStorage: ReadPackageFileAsync - File 'appsettings.keyvaluestorageconfiguration.json' does not exist in package 
07-15 16:26:35.758-04:00 W/Uno.Extensions.Storage.FileStorage: ReadPackageFileAsync - File 'appsettings.appconfig.json' does not exist in package 
07-15 16:26:35.760-04:00 W/Uno.Extensions.Storage.FileStorage: ReadPackageFileAsync - File 'appsettings.localizationconfiguration.json' does not exist in package 
07-15 16:26:35.762-04:00 W/Uno.Extensions.Storage.FileStorage: ReadPackageFileAsync - File 'appsettings.localizationsettings.json' does not exist in package 
07-15 16:26:35.764-04:00 W/Uno.Extensions.Storage.FileStorage: ReadPackageFileAsync - File 'appsettings.navigationconfiguration.json' does not exist in package 
07-15 16:26:35.796-04:00 I/Uno.Extensions.Navigation.Navigators.ContentControlNavigator: NavigateAsync - Starting Navigation - Navigator: ContentControlNavigator Request:  
07-15 16:26:35.805-04:00 W/Uno.Extensions.Navigation.Navigators.ContentControlNavigator: Show - Missing view for navigation path '' 
07-15 16:26:35.807-04:00 I/Uno.Extensions.Navigation.Navigators.ContentControlNavigator: CreateControlFromType - Creating control of type FrameView 
07-15 16:26:35.947-04:00 I/Uno.Extensions.Navigation.Navigators.FrameNavigator: NavigateAsync - Starting Navigation - Navigator: FrameNavigator Request: Main 
07-15 16:26:35.988-04:00 E/Microsoft.UI.Xaml.BrushTransition: The member BrushTransition.BrushTransition() is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.BrushTransition.BrushTransition.BrushTransition%28%29 
07-15 16:26:35.991-04:00 E/Microsoft.UI.Xaml.BrushTransition: The member TimeSpan BrushTransition.Duration is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.BrushTransition.TimeSpan%20BrushTransition.Duration 
07-15 16:26:35.994-04:00 E/Microsoft.UI.Xaml.Controls.ContentPresenter: The member BrushTransition ContentPresenter.BackgroundTransition is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Controls.ContentPresenter.BrushTransition%20ContentPresenter.BackgroundTransition 
The Bindable attribute is missing and the type [ElementsTechnician.Presentation.MainViewModel] is not known by the MetadataProvider. Reflection was used instead of the binding engine and generated static metadata. Add the Bindable attribute to prevent this message and performance issues.
07-15 16:26:36.140-04:00 I/Uno.Extensions.Navigation.Navigators.PanelVisiblityNavigator: CreateControlFromType - Creating control of type DirectConnectionControl 
07-15 16:26:36.185-04:00 E/Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings: The member ComboBoxTemplateSettings.DropDownContentMinWidth is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings.ComboBoxTemplateSettings.DropDownContentMinWidth 
07-15 16:26:36.220-04:00 E/Microsoft.UI.Xaml.Media.Animation.SplitCloseThemeAnimation: The member SplitCloseThemeAnimation.SplitCloseThemeAnimation() is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Media.Animation.SplitCloseThemeAnimation.SplitCloseThemeAnimation.SplitCloseThemeAnimation%28%29 
07-15 16:26:36.224-04:00 E/Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings: The member ComboBoxTemplateSettings.DropDownOffset is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings.ComboBoxTemplateSettings.DropDownOffset 
07-15 16:26:36.226-04:00 E/Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings: The member ComboBoxTemplateSettings.DropDownOpenedHeight is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Controls.Primitives.ComboBoxTemplateSettings.ComboBoxTemplateSettings.DropDownOpenedHeight 
07-15 16:26:36.228-04:00 E/Microsoft.UI.Xaml.Media.Animation.SplitCloseThemeAnimation: The member void Begin() is not implemented. For more information, visit https://aka.platform.uno/notimplemented#m=Microsoft.UI.Xaml.Media.Animation.SplitCloseThemeAnimation.void%20Begin%28%29

**Comment by jeromelaban** [comment]
Thanks for the question. 

Most of these warnings are going away in 5.3 and later, as they are already reported at compile time. The `ReadPackageFileAsync` warnings are being worked on https://github.com/unoplatform/uno.extensions/issues/2423

**Comment by ArchieCoder** [comment]
Cool! Closing.
